esi_leap/api/controllers/v1/node.py

```diff
@@ -64,15 +64,31 @@ def __init__(self, **kwargs):
 
 class NodesController(rest.RestController):
 
-    @wsme_pecan.wsexpose(NodeCollection)
-    def get_all(self):
+    @wsme_pecan.wsexpose(NodeCollection, wtypes.text,
+                         wtypes.text, wtypes.text)
+    def get_all(self, resource_class=None, owner=None,
+                lessee=None):
         context = pecan.request.context
 
+        if owner is not None:
+            owner = keystone.get_project_uuid_from_ident(context, owner)
```

Comment:
This won't quite work; that's not the aguments `keystone.get_project_uuid_from_ident` expects. It's worth double-checking.